common/jenkins-agents/python/docker/Dockerfile.rhel8

```diff
@@ -0,0 +1,29 @@
+FROM opendevstackorg/ods-jenkins-agent-base-centos7:latest
```

Comment:
I guess we should wait with merging this until we have published the new UBI8-based base image so that we can also add this image to the test suite.

Reply:
You can use https://hub.docker.com/layers/127012478/opendevstackorg/ods-jenkins-agent-base-ubi8/latest/images/sha256-8a0d86c6bff3758e3726b9722c4209dd4de948efdb48915cbc333267c6912e4c?context=explore now.

Reply:
Updated image reference and Dockerfile suffix.